autobot/telegram/objects/inlinekeyboard.py

```diff
@@ -62,7 +63,7 @@ class InlineKeyboardButton(BaseObject):
 
     """
     
-    __slots__ = ("text",
+    __slots__ = (
```

Comment:
I don't think "text" should have been removed here

Reply:
I thought __slots__ only only contain optional properties in our use case. Correct me if am wrong.

Reply:
when using `__slots__()` in python, if you don't 
include a slot for a variable, then that variable 
cannot be used, that is to say, whether they are optional or not as
long as they are supposed to be part of the class, they have to be defined in 
the `__slots__()`

Reply:
Alright. Thanks for the clarification. Let me correct it.